test/controllers/ApprovalControllerSpec.scala

```diff
@@ -159,6 +159,35 @@ class ApprovalControllerSpec extends WordSpec with Matchers with GuiceOneAppPerS
       }
     }
 
+    "the request is invalid with a DuplicateKeyError" should {
+
+      trait InvalidSaveTest extends Test {
+        val processError: ProcessError = toProcessErr(DuplicatePageUrl("4", "/feeling-bad"))
```

Comment:
Are processError and expectedError used?

Reply:
I'll sort out - and update the test headers that refer to bad request!

Reply:
done